package.json

```diff
@@ -3,14 +3,18 @@
     "version": "0.1.0",
     "private": true,
     "dependencies": {
+        "@aws-amplify/ui-react": "^4.3.4",
         "@testing-library/jest-dom": "^5.16.5",
         "@testing-library/react": "^13.4.0",
         "@testing-library/user-event": "^13.5.0",
         "@types/jest": "^27.5.2",
         "@types/node": "^16.18.4",
         "@types/react": "^18.0.25",
         "@types/react-dom": "^18.0.9",
+        "@types/react-router-dom": "^5.3.3",
```

Comment:
![image](https://user-images.githubusercontent.com/70910513/214614620-22cdf003-6104-43c0-9de3-9e21e0f3acce.png)

moved to make sure its not a types issue in router

Reply:
types was not the issue.